src/gensxg.c

```diff
@@ -313,6 +327,14 @@ static bool validate(const Options* opt) {
   return valid;
 }
 
+static bool validate(const Options* opt) {
+  bool valid = validate_common_options(opt);
+  if (!opt->integrity_mode) {
+    valid = valid & validate_generator_options(opt);  /* & should not be &&. */
```

Comment:
The `/* ... */` comment was meant just to be a message to you and thus can be omitted.

Reply:
removed.